src/drivers/intel/hda/hda-dma.c

```diff
@@ -979,8 +979,6 @@ const struct dma_ops hda_host_dma_ops = {
 	.start			= hda_dma_start,
 	.stop			= hda_dma_stop,
 	.copy			= hda_dma_host_copy,
-	.pause			= hda_dma_pause,
-	.release		= hda_dma_release,
```

Comment:
"hda_dma_pause/release declared but not used"?

Reply:
sorry dont get it? it is used in link DMA ops

Reply:
> sorry dont get it? it is used in link DMA ops

Oh, makes sense then, maybe better to rename them to hda_dma_link_pause/release().